docs/content/manual/manual.yml

```diff
@@ -908,14 +908,18 @@ sections:
 
           For any filter `f`, `map(f)` will run that filter for each
           element of the input array, and return the outputs in a new
-          array. `map(.+1)` will increment each element of an array of numbers.
-
-          Similarly, `map_values(f)` will run that filter for each element,
-          but it will return an object when an object is passed.
+          array. For example, `map(.+1)` will increment each element
+          of an array of numbers, and `map(.,.)` will duplicate each
+          element of the input array.
 
-          `map(f)` is equivalent to `[.[] | f]`. In fact, this is how
-          it's defined. Similarly, `map_values(f)` is defined as `.[] |= f`.
+          `map(f)` is equivalent to `[.[] | f]`. In fact, this is how it's defined.
 
+          `map_values(f)` is similarly defined as `.[] |= f`.  This
+          essentially means that each value, $v, in the input array or
+          object is replaced by `first($v|f)`, it being understood
+          that if the input is an object, then keys for which this
```

Comment:
Array values for which `f` returns `empty` also get deleted.

I liked the original text better.  Staring at the definitions of `map/1` and `map_values/1` the difference between them isn't quite obvious, but the original text here makes it obvious: `map/1` always returns an array, while `map_values/1` returns an array if its input was an array, or an object if its input was an object.

Reply:
Tx. Hopefully the subsequent commit addresses all these concerns.  It is more systematic about the various possibilities.